src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -66,24 +66,8 @@ UserStore thisUserStore = (UserStore) request.getAttribute("userStore");
     <a href="/activity.jsp">Activity</a>
   </nav>
 
-<<<<<<< HEAD
-<!-- This is the test UI for the pop up box -->
-    <button onclick="getMessageId()">Try it</button>
-
-    <script>
-    function getMessageId() {
-        var message = "/* get the current message or current message ID here */";
-        var previousMessage = prompt("Please enter your new message", "/* get the content of the current message here */");
-        console.log("/* put the current message id here for debugging purposes */");
-    }
-    //creating function in which will take action to delete message
-    
-    </script>s
 
-<!-- End of the test UI for the pop up box -->
 
-=======
->>>>>>> b7c16b5a11ed7d6aa16c5513fb47e3bbb67bfa50
```

Comment:
Is this merge conflict fixed?

Reply:
Yeah, I think so. It was for the edit button functionality, so I took out the temporary stuff I had there and replaced it with your changes.